src/Components/HourlyWeatherCard/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { FC } from "react";
+import { useWeatherData } from "../../Context";
+
+const HourlyWeatherCard: FC = () => {
+  const data = useWeatherData();
+  const date = new URLSearchParams(window.location.search).get("date");
+  console.log(date);
+  console.log(data);
```

Comment:
Don't push console logs on remote

Reply:
Just Write Now testing Something I will Remove it later.